python/ray/experimental/state.py

```diff
@@ -569,3 +638,27 @@ def workers(self):
                 "stdout_file": worker_info[b"stdout_file"].decode("ascii")
             }
         return workers_data
+
+    def job_length(self):
```

Comment:
Let's name this `_job_length` so that it isn't exposed to the user.

Also, Please add a docstring explaining precisely what this is measuring.

Also, note that there is a value in Redis already `'redis_start_time'` which is set to the time when the cluster was initiated. Not sure if we want to use this, but it's there (note that it's different from what is currently being used which is presumably the time the first task started).